src/commander/commander_btc.c

```diff
@@ -62,6 +62,7 @@ static commander_error_t _btc_pub_xpub(const BTCPubRequest* request, PubResponse
             return COMMANDER_ERR_GENERIC;
         }
         char title[100] = {0};
+        int n_written;
```

Comment:
why not inline? principle of smallest scope.

Reply:
You can't allocate stack space within a `switch` statement!